src/styles/overrides/_se-events.scss

```diff
@@ -31,6 +31,7 @@
     list-style-type: none;
     margin: 0;
     padding: 0;
+    padding-top: $padding-small;
```

Comment:
Recommending removing `padding-top` and rewriting the `padding `property on line 33 in the following: `padding: $padding-small 0 0 0;`